app/app/main.py

```diff
@@ -80,55 +84,127 @@ def validate_extension(filename):
     )
  
 
+def remove_folder(path,tdate):
+
+        if not shutil.rmtree(path):
+                logger.info(f"---{tdate}---{path} folder is removed successfully")
+        else:
+                logger.info(f"---{tdate}---Unable to delete the {path}")
+
+
+def remove_file(path,tdate):
+
+           if not os.remove(path):         
+                logger.info(f"---{tdate}---{path} file is removed successfully")
+           else:           
+                logger.info(f"---{tdate}---Unable to delete the {path}")
+
 
+def get_file_or_folder_age(path):
+
+        # getting modified time of the file/folder
+        # time will be in seconds
+        mtime = os.stat(path).st_mtime
+
+        return mtime
```

Comment:
why not just return `os.stat(path).st_mtime`?

Reply:
we can but what's wrong with this ?

Reply:
The assignment does not add anything, and it does not improve readability. Considering how short this line is it could be called directly in the code.

Reply:
okay